mod/05_standard/set/self/account_links.rb

```diff
@@ -5,8 +5,9 @@
     #ENGLISH
     links = []
     if Auth.signed_in?
-      links << link_to_page( Auth.current.name, nil, :id=>'my-card-link' )
+      links << build_link( Auth.current.cardname, nil, :id=>'my-card-link' )
```

Comment:
Ah, yes, that would be fine.  I could add that pretty easily.

Well, link_to is what the rails method that link_to_X was using before.  I think it may be used, maybe quite a bit.  Then 'format_link' may be more accurate than build_link.  We could bring back the other method name (link_to_p...) as it was but use this API (with suggested change with :text).  I'll add whatever you like best.  I'm even ok with build_link.